src/rulesets/oas2/index.json

```diff
@@ -73,6 +73,19 @@
         "function": "truthy"
       }
     },
+    "oas2-parameter-description": {
+      "description": "Parameter objects should have a `description`.",
+      "recommended": false,
+      "type": "style",
+      "given": "$..[?(@parentProperty === 'parameters' && @.in)]",
```

Comment:
Over a call we tried to simplify it and use some sort of "union" but weren't able to find the right syntax.